### PR TITLE
Work with v9 multiple "events".

### PR DESCRIPTION
**Pull request by igloo32**

### CODE REVIEW

**Comment by igloo32** [comment]
Updated to work with v9 multiple "events". Added option to Combine same rates in a day. Added option to highlight cheapest future rate. Added multiplier for display in pence/pounds

File: octopus-energy-rates-card.js [hunk 1]
```diff
@@ -231,12 +324,12 @@ class OctopusEnergyRatesCard extends HTMLElement {
     }
 }
 
-customElements.define('octopus-energy-rates-card', OctopusEnergyRatesCard);
```
Comment (WhyAydan) [inline octopus-energy-rates-card.js:234]: Any reason for it being called OctopusEnergyRatesCardMv2?

**Review by WhyAydan — COMMENTED** [review]
Works for me other than the commented line

File: octopus-energy-rates-card.js [hunk 2]
```diff
@@ -231,12 +324,12 @@ class OctopusEnergyRatesCard extends HTMLElement {
...
+customElements.define('octopus-energy-rates-card', OctopusEnergyRatesCardMv2);
```
Comment (WhyAydan) [inline octopus-energy-rates-card.js:327]: Any reason for it being called OctopusEnergyRatesCardMv2?

**Review by WhyAydan — COMMENTED** [review]
*Commented on the wrong side*

File: octopus-energy-rates-card.js (hunk 2 above)
Comment (igloo32) [reply octopus-energy-rates-card.js:327]: I've been running it side by side with the origional so renamed it in my local one, just missed removing it when checking in the change

Comment (WhyAydan) [reply octopus-energy-rates-card.js]: Ah perfect! Working great on mine.

**Comment by bwduncan** [comment]
Hooray! Thank you so much :grin: 

It looks like the pastEntity and futureEntity attributes are not optional. The card doesn't load without them. I'm not really interested in yesterday's rates, but this should at least be reflected in the documentation. I'm happy to make a PR to do that, but I'm leaving this comment first in case I'm wrong.

**Pull request merged**